make-options.json

```diff
@@ -151,6 +151,7 @@
         "PowerShellOnTargetMachinesV3",
         "PublishBuildArtifactsV1",
         "PublishCodeCoverageResultsV1",
+        "PublishCodeCoverageResultsV2",
```

Comment:
Please update task version to latest:
![image](https://user-images.githubusercontent.com/92331194/208383349-41a776ac-d044-406f-b247-27aeda6d9022.png)

Reply:
updated